app/src/main/java/edu/wisc/ece/pinpoint/MainActivity.java

```diff
@@ -128,17 +122,17 @@ public void handleOnBackPressed() {
             showView(R.id.content_view);
         });
 
-        Handler handler = new Handler(Looper.getMainLooper());
-        handler.postDelayed(() -> {
-            PeriodicWorkRequest saveRequest =
-                    new PeriodicWorkRequest.Builder(PinNotificationActivity.class, 16,
-                            TimeUnit.MINUTES)
-                            // Constraints
-                            .build();
-
-            WorkManager work = WorkManager.getInstance(getApplicationContext());
-            work.enqueue(saveRequest);
-        }, 1);
+//        Handler handler = new Handler(Looper.getMainLooper());
```

Comment:
Is this intended to be pushed to main? I know we're troubleshooting notifs but wanna make sure this is intentional

Reply:
Yeah we're exceeding our free quota by a ton due to these notifs on everyone's phones so we need to stop them for the time being until they get fixed.